src/features/content-explorer/content-explorer/ContentExplorerFolderTreeBreadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl } from 'react-intl';
+
+import Button from '../../../components/button';
+import DropdownMenu from '../../../components/dropdown-menu/DropdownMenu';
+import IconBreadcrumbArrow from '../../../icons/general/IconBreadcrumbArrow';
+import IconFolderTree from '../../../icons/general/IconFolderTree';
+import { Menu, MenuItem } from '../../../components/menu';
+
+import { FoldersPathPropType } from '../prop-types';
+import messages from '../messages';
+import { bdlGray50 } from '../../../styles/variables';
+
+const ContentExplorerFolderTreeBreadcrumbs = ({
+    foldersPath,
+    intl: { formatMessage },
+    isFolderTreeButtonDisabled = false,
+    numTotalItems,
+    onBreadcrumbClick,
+}) => (
+    <div className="content-explorer-folder-tree-container">
+        <DropdownMenu>
+            <Button
+                aria-label={formatMessage(messages.clickToGoBack)}
+                className="content-explorer-folder-tree-button"
+                type="button"
+                title="file path"
+                isDisabled={isFolderTreeButtonDisabled}
+            >
+                <IconFolderTree height={18} width={18} />
+            </Button>
+
+            <Menu>
+                {foldersPath.map((folder, i) => (
+                    <MenuItem
+                        data-testid="breadcrumb-lnk"
+                        key={folder.name}
+                        onClick={event => onBreadcrumbClick(i, event)}
+                    >
+                        {folder.name}
+                    </MenuItem>
+                ))}
+            </Menu>
+        </DropdownMenu>
+
+        <IconBreadcrumbArrow className="icon-breadcrumb-main" height={7.5} width={5} color={bdlGray50} />
+
+        <span className="folder-title-breadcrumb-text" title={foldersPath[foldersPath.length - 1].name}>
```

Comment:
This was the intention because the current breadcrumb component from ContentExplorerBreadcrumbs.js uses the title prop over the  folder names as well. I thought that making it a Tooltip might be an overuse of Tooltips on this modal, but I am also open to switching it to a Tooltip if we think that is best

Reply:
I spoke to a designer and we agreed to just remove the tooltips in general as they are redundant and not entirely helpful.